docs/source/author/writing_guide.rst

```diff
@@ -227,47 +227,62 @@ destination of the link.
-``content/`` directory (see examples below).
+.. note::
 
-Link to Page Within Website (relative link - Verto feature)
-------------------------------------------------------------------------------
+  This type of link requires knowledge of project URL routing, required URL
+  values, and the `Django URL template tag <https://docs.djangoproject.com/en/1.11/ref/templates/builtins/#url>`__.
+  If you are unsure on how to create a link of this type, please contact us
+  and we will be happy to help.
 
-You can refer to a page by writing the page name with ``.html`` at the end.
-The name of a file is defined by it's slug in the configuration files, but
-it helps to have knowledge of the resulting URL path for a file.
-See the examples below:
+Links to pages within the website use the Django ``url`` template tag.
+In order to create an link, use the values that would be given to the template tag.
+See the example below for an example.
+
+This following Markdown:
 
 .. code-block:: none
 
-  Check out [binary numbers](topics/binary-numbers.html).
-  Check out the [about page](about.html).
+  See [challenge 4.1]('topics:programming_challenge' 'kidbots' 'check-if-divisor').
+
+results in the following HTML:
 
-`Click here to read the documentation on how to create a relative link`_.
+.. code-block:: html
 
-Link to Heading on Page Within Website
+  See <a href="/en/topics/kidbots/programming/check-if-divisor">challenge 4.1</a>.
+
+Currently linking to a heading within the page is not supported (or recommended
+as heading text can change easily).
+
+Link to resource page with pre-selected options
 ------------------------------------------------------------------------------
 
-You can refer to a subsection on a page by following the same syntax as above
-and then adding the subsection name at the end with a ``#`` separator.
-All headers are subsections that have a link that can be linked to (called an
-anchor link).
-The anchor link can be determined by converting the header name to lowercase,
-with spaces replaced with dashes, and punctuation removed.
-In cases where duplicate headings exist on the same page, a number is appended
-on the end of the anchor link.
+.. note::
+
+  This type of link requires knowledge of project URL routing, required URL
+  values, and the `Django URL template tag <https://docs.djangoproject.com/en/1.11/ref/templates/builtins/#url>`__.
+  If you are unsure on how to create a link of this type, please contact us
+  and we will be happy to help.
+
+This link uses the same syntax as internal links, with appended query parameters.
+Each parameter should be a resource option with its desired value.
+Values given as query parameters override the default values for each resource.
+
+This following Markdown:
 
 .. code-block:: none
 
-  Please [contact us](about/index.html#contact).
+  [Little Red Riding Hood cards]('resources:resource' 'sorting-network-cards'?type=riding_hood)
```

Comment:
pls clarify,  the template for relative-link doesn't inlcude params, is `type=riding_hood` just included as part of the path? in which case, how is this parsed (there are quotes around `sorting-network-cards`, what does this mean). Basically, what is this magic?